core/src/main/java/org/apache/calcite/rel/RelDistributions.java

```diff
@@ -80,6 +80,10 @@ public static RelDistribution range(Collection<? extends Number> numbers) {
     return RelDistributionTraitDef.INSTANCE.canonize(trait);
   }
 
+  public static RelDistribution with(RelDistribution.Type type, ImmutableIntList keys) {
```

Comment:
maybe `of()` to be consistent with `RelCollatioins`?

Reply:
+1

Reply:
Renamed to `of`